### DISCUSSION TITLE
KoboldCPP-v1.63.yr1-ROCm Error

### DISCUSSION THREAD

**Discussion by Pablo4096** [YellowRoseCx/koboldcpp-rocm#42] [General]
In the new version, generation is interrupted by the "Generating (30 / 512 tokens)exception: access violation reading 0x0000000000000000" 
I have a rx6600, I have changed the default settings, temperture, context size and I use Microstat v2

**Comment by dya3506** [comment]
Uncheck "Use QuantMatMul(mmq)"

**Reply by Yama-K** [reply]
I wish it helped, still experiencing "exception: access violation reading 0x0000000000000001" on 6800 XT

**Reply by Yama-K** [reply]
However, switching to vulkan fixes this, not that it's a real fix if you want to use rocm

**Comment by Pablo4096** [comment]
Thanks, it helped. Now it works without any problem.
